package.json

```diff
@@ -29,7 +29,7 @@
   },
   "homepage": "https://github.com/Eppo-exp/node-server-sdk#readme",
   "dependencies": {
-    "@eppo/js-client-sdk-common": "^4.15.1"
```

Comment:
I'm recalling that we had decided in the past to avoid using loose semver notations (`~` and `^`) because we run the risk that if we release a change in common that is incompatible with one of the SDKs, that it might automatically get picked up and cause issues?

Reply:
Yes good catch this is safer